_pages/oidc.md

```diff
@@ -91,23 +91,37 @@ https://idp.int.identitysandbox.gov/openid_connect/authorize?
 
 ### Request Parameters
 
-* **acr_values**
-  The Authentication Context Class Reference values used to specify the IAL (Identity Assurance Level) of an account, either IAL1, or IAL2. This and the `scope` determine which [user attributes]({{ site.baseurl }}/attributes/) will be available in the [user info response](#user-info-response). The possible parameter values are:
-    - `http://idmanagement.gov/ns/assurance/ial/1`
-    - `http://idmanagement.gov/ns/assurance/ial/2`
-
 
-#### Level of Assurance (LOA)
 
-<div class="usa-alert usa-alert-warning">
-  <div class="usa-alert-body">We strongly recommend using IAL for the identity proofing process. The concept of Level of Assurance (LOA) is retired by the NIST 800-63-3 digital identity guidelines, and support by login.gov for LOA requests is deprecated.
-  </div>
-</div>
-
-  The authentication request can specify LOA levels 1 and 3 with one of these values as the `acr_value`:
-  >  - `http://idmanagement.gov/ns/assurance/loa/1`
-  >  - `http://idmanagement.gov/ns/assurance/loa/3`
-<br>
+* **acr_values**
+  The Authentication Context Class Reference requests can be used to specify the IAL (Identity Assurance Level) or the AAL (Authentication Assurance Level) for the user. These and the `scope` determine which [user attributes]({{ site.baseurl }}/attributes/) will be available in the [user info response](#user-info-response).
+
+  Multiple values can be joined with a space (before being URI-escaped in the final URL)
+
+  #### IAL Values
+  An IAL value must be specified.
+
+    - **`http://idmanagement.gov/ns/assurance/ial/1`**
+        Basic identity assurance, does not require identity verification (this is the most common value).
+    - **`http://idmanagement.gov/ns/assurance/ial/2`**
+        Requires that the user has gone through identity verification
+    - **`http://idmanagement.gov/ns/assurance/ial/2?strict=true`**
+        Requires that the user has gone through identity verification, including a "liveness" check
+
+  #### AAL Values
+    - **`http://idmanagement.gov/ns/assurance/aal/2`**
```

Comment:
We want to reserve /aal/2 for **strict AAL2** (currenlty not supported.

Propose: 

We default to require a user to be authenticated with a second factor.

**`http://idmanagement.gov/ns/assurance/aal/3`**
        This specifies that a user has been authenticated with a crytographically secure method, such as WebAuthn or using a PIV/CAC.

Reply:
Ok! Removed aal2, updated it in 256b650bc59c3338c38090a96f0ee90d4bdc0bcc